submissions/Anatolii-Petrenko/tiny-js-world/index.js

```diff
@@ -0,0 +1,66 @@
+import { print } from "./js/lib.js";
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Anatolii-Petrenko/a-tiny-JS-world
+   Web app: https://anatolii-petrenko.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+  species: "human",
+  name: "Antony",
+  gender: "male",
+  hands: 2,
+  legs: 2,
+  saying: "Hi Bro!",
+};
+
+const woman = {
+  species: "human",
+  name: "Jeeny",
+  gender: "female",
+  hands: 2,
+  legs: 2,
+  saying: "Hello! How it's going?",
+};
+
+const dog = {
+  species: "dog",
+  name: "Toby",
+  gender: "male",
+  hands: 0,
+  legs: 4,
+  saying: "Woof-Woof!",
+};
+
+const cat = {
+  species: "cat",
+  name: "Bella",
+  gender: "female",
+  hands: 0,
+  legs: 4,
+  saying: "Meow!",
+};
+
+const catWoman = {
+  species: "human",
+  name: "JeenyCat",
+  gender: "female",
+  hands: 2,
+  legs: 2,
+  saying: cat.saying,
```

Comment:
Notice that this will create a copy of the saying. So when we change a cat's saying, cat-woman won't get updated.